src/components/InForm/InForm.tsx

```diff
@@ -10,73 +10,64 @@ import {
   mergeStyleSets,
   Modal,
 } from "@fluentui/react";
-import { makeStyles } from "@fluentui/react-components";
-import React, { ChangeEvent, FormEvent, useState } from "react";
-import { PeoplePicker } from "../PeoplePicker/PeoplePicker";
+import React, { ChangeEvent, FormEvent, useContext, useState } from "react";
```

Comment:
Remove global React import

Reply:
Updated as part of review of PeoplePicker PR